.github/workflows/merge.yml

```diff
@@ -593,6 +593,297 @@ jobs:
           log_file_prefix: ant_test_logs_e2e
           build: true
 
+  pointer:
```

Comment:
As per the scratchpad tests, we can consider having these in their own file.

However, purely optional.